src/texture_override.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+Minetest
+Copyright (C) 2010-2013 celeron55, Perttu Ahola <celeron55@gmail.com>
```

Comment:
celeron55 didn't write this, nor did he do so between 2010 and 2013

Reply:
What should be there instead? I figured I may as well leave the notice consistent with the rest of the code, but I can change the date/credit or remove the notice as well if need be.

Reply:
It should be 

>  Copyright (C) 2020 Df458